### DISCUSSION TITLE
How to redefine the behavior of modifier keys?

### DISCUSSION THREAD

**Discussion by MrPandir** [Chatterino/chatterino2#5301] [Q&A]
How to change the actions of the keys `command` and `command + alt`?

https://github.com/Chatterino/chatterino2/assets/137798474/a76a0d67-b125-47a6-a023-05adfaea55a6

**Comment by pajlada** [comment]
It's not possible ~~, the video doesn't work for me~~. What is it that you're trying to do? 

EDIT: I just had to complain & the video started working

**Comment by MrPandir** [comment]
> It's not possible. What is it that you're trying to do?

I would like to assign these keys to modifiers for other actions. For example, assign the action `Open mod view in browser` on `command + alt + M`. For me, this is a very comfortable combination. :ok_hand:
If I do this now, every time I press the combination of `command + alt + M`, view mod opens, and I also get a menu for changing the size and moving `split` (as in the video). Therefore, it would be glorious if the combination of `command` and `command + alt` added to `Settings > Hotkeys`.

If it is possible to implement it, please transform it into Issue.

**Reply by pajlada** [reply]
I think this would most likely live outside of hotkeys, but I'm ok with these modifiers being changeable
